scripts/buildPlatforms/buildFigma.ts

```diff
@@ -30,7 +38,7 @@ export const buildFigma = (buildOptions: ConfigGeneratorOptions): void => {
   }
   //
   for (const {filename, source, include} of themes) {
-    if (['light', 'dark'].includes(filename)) {
+    if (!['light-high-contrast', 'dark-high-contrast', 'dark-dimmed'].includes(filename)) {
```

Comment:
What about color blind themes?

Reply:
This is just disabling them to test if they are problematic, they are. It is just a temp step.